transcom/mymove
# B-20445: Add Marine Corps BOS to Move Search and BOS filter in work queues

## [Agility ticket B-20445](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20445)

## Testing Steps
1. Create a new use that is Marine Corps
![Screenshot 2024-08-28 at 1 30 41 PM](https://github.com/user-attachments/assets/d2332872-31e9-4438-b32c-38b1248c8236)

Note: While creating, you may need to take note of postal codes to GBLOC conversions (https://dp3.atlassian.net/wiki/download/attachments/2279735297/postal_code_to_gblocs_202311091157.csv?api=v2)

2. Create a HHG move/shipment so that we can check the queues for office users.
3. After the move/shipment is sent to the SC for approval, sign in as an SC under the appropriate GBLOC.
4. For the initial queue, there should be a filter for branch.. select Marine Corps from that list and you should see only Marine Corps moves.
5. Select that move and approve as SC.
6. As TOO/TIO verify that you are able to see the created move when you filter by Marine Corps.
7. As TOO make sure you approve the move/shipment so that it will be available to Prime.
8. As Prime verify that you are able to see the created move when you filter by Marine Corps.

___

## ANDI screenshot
![Screenshot 2024-08-28 at 1 47 34 PM](https://github.com/user-attachments/assets/2b43ba34-95ff-45a5-a0cc-46b5e8e96b61)
